app/Friend.php

```diff
@@ -23,8 +29,29 @@ public function markVerified()
         $this->save();
     }
 
+    public function markBlacklisted()
+    {
+        $this->blacklisted = true;
+        $this->save();
+        event(new FriendWasBlacklisted($this));
+    }
+
     public function scopeVerified($query)
     {
-        return $query->where('is_verified', true);
+        return $query->where([
+            'is_verified' => true,
+            'blacklisted' => false
+        ]);
+    }
+
+    public function getStatusAttribute()
+    {
```

Comment:
I'd probably make these three strings be constants on the class, since we're attributing quite a bit of significance to them.

Also, even though it's not as correct English, I'd probably drop the dash from `un-verified` so they're all one word.